app-benchmarks/sysbench/sysbench-1.0.11.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils
+
+DESCRIPTION="System performance benchmark"
+HOMEPAGE="https://github.com/akopytov/sysbench"
+SRC_URI="https://github.com/akopytov/sysbench/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="aio mysql postgres test"
+
+RDEPEND="aio? ( dev-libs/libaio )
+	mysql? ( virtual/libmysqlclient )
+	postgres? ( dev-db/postgresql:= )"
+DEPEND="${RDEPEND}
+	app-editors/vim-core
+	dev-lang/luajit:=
+	dev-libs/concurrencykit
+	dev-libs/libxslt
+	sys-devel/libtool:=
+	virtual/pkgconfig
+	test? ( dev-util/cram )"
+
+src_prepare() {
+	default
+
+	sed -i -e "/^htmldir =/s:=.*:=/usr/share/doc/${PF}/html:" doc/Makefile.am || die
+
+	# remove bundled libs
+	rm -r third_party/luajit/luajit third_party/concurrency_kit/ck third_party/cram || die
+
+	./autogen.sh || die
+}
+
+src_configure() {
+	local myeconfargs=(
```

Comment:
Why can't we just pass these arguments directly to econf?

Reply:
@titanofold, yes we could, but each lines must be then backslashed which looks a bit uglier I suppose. This approach seems like a standard for most packages, so I would keep it like this.